io/src/vlp_grabber.cpp

```diff
@@ -139,54 +147,64 @@ pcl::VLPGrabber::toPointClouds (HDLDataPacket *dataPacket)
       {
         if (current_sweep_xyz_->size () > 0)
         {
-          current_sweep_xyz_->is_dense = current_sweep_xyzi_->is_dense = false;
+          current_sweep_xyz_->is_dense = current_sweep_xyzrgb_->is_dense = current_sweep_xyzi_->is_dense = false;
           current_sweep_xyz_->header.stamp = velodyne_time;
+          current_sweep_xyzrgb_->header.stamp = velodyne_time;
           current_sweep_xyzi_->header.stamp = velodyne_time;
           current_sweep_xyz_->header.seq = sweep_counter;
+          current_sweep_xyzrgb_->header.seq = sweep_counter;
           current_sweep_xyzi_->header.seq = sweep_counter;
 
           sweep_counter++;
 
           HDLGrabber::fireCurrentSweep ();
         }
         current_sweep_xyz_.reset (new pcl::PointCloud<pcl::PointXYZ> ());
+        current_sweep_xyzrgb_.reset (new pcl::PointCloud<pcl::PointXYZRGBA> ());
```

Comment:
In order to change name of this variable, It need to change base class.
I think It would be better to separate it into other pull requests.
What do you think?

BTW, Sorry for late reply.
I'm planning to start fix this pull request in a few days. Thanks,